### PR TITLE
Feature/binary ts

### PR DESCRIPTION
**Pull request by rma-bryson**

### CODE REVIEW

File: cwms-radar-client/src/main/java/mil/army/usace/hec/cwms/radar/client/controllers/BinaryTimeSeriesEndpointInput.java [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+        return new Delete(timeSeriesId, officeId);
+    }
+
+    public static final class GetAll extends EndpointInput {
+        static final String OFFICE_QUERY_PARAMETER = "office";
+        static final String BEGIN_QUERY_PARAMETER = "begin";
+        static final String END_QUERY_PARAMETER = "end";
+        static final String PAGE_QUERY_PARAMETER = "page";
+        static final String PAGE_SIZE_QUERY_PARAMETER = "page-size";
+        static final String MIN_ATTRIBUTE_QUERY_PARAMETER = "min-attribute";
```
Comment (adamkorynta) [inline cwms-radar-client/src/main/java/mil/army/usace/hec/cwms/radar/client/controllers/BinaryTimeSeriesEndpointInput.java:61]: Need to remove attributes

File: cwms-radar-client/src/main/java/mil/army/usace/hec/cwms/radar/client/controllers/BinaryTimeSeriesEndpointInput.java [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+            return httpRequestBuilder.addQueryParameter(REPLACE_ALL_PARAMETER, Boolean.toString(replaceAll))
+                    .addQueryHeader(ACCEPT_QUERY_HEADER, ACCEPT_HEADER_V2);
+        }
+    }
+
+    public static final class Delete extends EndpointInput {
+        static final String OFFICE_QUERY_PARAMETER = "office";
+        static final String BEGIN_PARAMETER_QUERY = "begin";
+        static final String END_PARAMETER_QUERY = "end";
+        static final String MIN_ATTRIBUTE_QUERY_PARAMETER = "min-attribute";
```
Comment (adamkorynta) [inline cwms-radar-client/src/main/java/mil/army/usace/hec/cwms/radar/client/controllers/BinaryTimeSeriesEndpointInput.java:156]: remove attributes

File: cwms-radar-client/src/main/java/mil/army/usace/hec/cwms/radar/client/controllers/BinaryTimeSeriesEndpointInput.java [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+                    .addQueryParameter(BINARY_TYPE_MASK_QUERY_PARAMETER, binaryTypeMask)
+                    .addQueryParameter(BEGIN_PARAMETER_QUERY, beginString)
+                    .addQueryParameter(END_PARAMETER_QUERY, endString)
+                    .addQueryParameter(MIN_ATTRIBUTE_QUERY_PARAMETER, minAttributeString)
+                    .addQueryParameter(MAX_ATTRIBUTE_QUERY_PARAMETER, maxAttributeString)
+                    .addQueryHeader(ACCEPT_QUERY_HEADER, ACCEPT_HEADER_V1);
+        }
+    }
+
+    public enum Mode {
```
Comment (adamkorynta) [inline cwms-radar-client/src/main/java/mil/army/usace/hec/cwms/radar/client/controllers/BinaryTimeSeriesEndpointInput.java:217]: Remove Mode

File: cwms-radar-model/src/main/java/mil/army/usace/hec/cwms/radar/client/model/BinaryTimeSeries.java [hunk 4]
```diff
@@ -0,0 +1,292 @@
...
+    @JsonProperty("interval-offset")
+    private Long intervalOffset = null;
+
+    @JsonProperty("time-zone")
+    private String timeZone = null;
+
+    /**
+     * Version type specifies the type of timeseries response to be received. Can be max aggregate or single version. Max aggregate cannot be run if version date field is specified.
+     */
+    public enum DateVersionTypeEnum {
```
Comment (adamkorynta) [inline cwms-radar-model/src/main/java/mil/army/usace/hec/cwms/radar/client/model/BinaryTimeSeries.java:59]: This should be the same DateVersionTypeEnum used in TextTimeSeries and TimeSeries. Should refactor them all out into an external class.

File: cwms-radar-model/src/main/java/mil/army/usace/hec/cwms/radar/client/model/BinaryTimeSeriesRow.java [hunk 5]
```diff
@@ -0,0 +1,281 @@
...
+
+    @JsonProperty("filename")
+    private String filename = null;
+
+    @JsonProperty("dest-flag")
+    private Integer destFlag = null;
+
+    @JsonProperty("binary-value")
+    @Valid
+    private List<byte[]> binaryValue = new ArrayList<>();
```
Comment (adamkorynta) [inline cwms-radar-model/src/main/java/mil/army/usace/hec/cwms/radar/client/model/BinaryTimeSeriesRow.java:58]: This is just `byte[]` not a list `List`

**Review by adamkorynta — CHANGES_REQUESTED** [review]

File: cwms-radar-client/src/main/java/mil/army/usace/hec/cwms/radar/client/controllers/BinaryTimeSeriesEndpointInput.java (hunk 1 above)
Comment (rma-bryson) [reply cwms-radar-client/src/main/java/mil/army/usace/hec/cwms/radar/client/controllers/BinaryTimeSeriesEndpointInput.java]: removed

File: cwms-radar-client/src/main/java/mil/army/usace/hec/cwms/radar/client/controllers/BinaryTimeSeriesEndpointInput.java (hunk 2 above)
Comment (rma-bryson) [reply cwms-radar-client/src/main/java/mil/army/usace/hec/cwms/radar/client/controllers/BinaryTimeSeriesEndpointInput.java]: removed

File: cwms-radar-client/src/main/java/mil/army/usace/hec/cwms/radar/client/controllers/BinaryTimeSeriesEndpointInput.java (hunk 3 above)
Comment (rma-bryson) [reply cwms-radar-client/src/main/java/mil/army/usace/hec/cwms/radar/client/controllers/BinaryTimeSeriesEndpointInput.java]: removed

File: cwms-radar-model/src/main/java/mil/army/usace/hec/cwms/radar/client/model/BinaryTimeSeries.java (hunk 4 above)
Comment (rma-bryson) [reply cwms-radar-model/src/main/java/mil/army/usace/hec/cwms/radar/client/model/BinaryTimeSeries.java]: Refactored this out

**Pull request merged**